content/millennium/r4/base/individuals/practitioner.md

```diff
@@ -19,31 +19,33 @@ Not all practitioners have access to the EHR but can be referenced by other reso
 
 The following fields are returned if valued:
 
-* [Practitioner id](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
-* [Identifiers/Aliases such as NPI and DEA](https://hl7.org/fhir/r4/practitioner-definitions.html#Practitioner.identifier){:target="_blank"}
-* [Active (true/false)](https://hl7.org/fhir/r4/practitioner-definitions.html#Practitioner.active){:target="_blank"}
+* [Practitioner ID](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Identifier](https://hl7.org/fhir/r4/practitioner-definitions.html#Practitioner.identifier){:target="_blank"}
+* [Active](https://hl7.org/fhir/r4/practitioner-definitions.html#Practitioner.active){:target="_blank"}
 * [Name](https://hl7.org/fhir/r4/practitioner-definitions.html#Practitioner.name){:target="_blank"}
-* [Telecom Information (secure email and phone)](https://hl7.org/fhir/r4/practitioner-definitions.html#Practitioner.telecom){:target="_blank"}
-* [Address (Provider and System Authorization Only)](https://hl7.org/fhir/r4/practitioner-definitions.html#Practitioner.address){:target="_blank"}
+* [Telecom](https://hl7.org/fhir/r4/practitioner-definitions.html#Practitioner.telecom){:target="_blank"}
+* [Address](https://hl7.org/fhir/r4/practitioner-definitions.html#Practitioner.address){:target="_blank"}
 * [Gender](https://hl7.org/fhir/r4/practitioner-definitions.html#Practitioner.gender){:target="_blank"}
 * [Qualification](https://hl7.org/fhir/r4/practitioner-definitions.html#Practitioner.qualification){:target="_blank"}
 * [Is Physician Extension](#extensions)
 
-## Terminology Bindings
+<%= disclaimer %>
 
-<%= terminology_table(:practitioner, :r4) %>
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
 
-## Extensions
+## Terminology Bindings
 
-* [Is Physician]
+<%= terminology_table(:practitioner, :r4) %>
 
-### Custom Extensions
+## Custom Extensions
 
 All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/StructureDefinition/{id}`
 
- ID                    | Value\[x] Type                                              | Description
------------------------|----------------|------------------------------------------------------------------------------------------------
- `is-physician`        | [`Boolean`](https://hl7.org/fhir/r4/datatypes.html#boolean) | Indication of whether the provider is a physician or not.
+| ID             | Value\[x] Type                                              | Description                                                   |
+|----------------|-------------------------------------------------------------|---------------------------------------------------------------|
+| `is-physician` | [`Boolean`](https://hl7.org/fhir/r4/datatypes.html#boolean) | Indication of whether the practitioner is a physician or not. |
```

Comment:
You made the `boolean` link at the end of the markdown file - you can use it here